tests/org.jboss.tools.maven.ui.bot.test/META-INF/MANIFEST.MF

```diff
@@ -19,3 +19,4 @@ Require-Bundle: org.eclipse.core.runtime,
 Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Vendor: Red Hat
+Export-Package: org.jboss.tools.maven.ui.bot.test

```

Comment:
I would prefer not to export test package because you need to reuse some methods. Isn't there a better way we are reusing reddeer plugins implementation?

Reply:
I am not aware of any other way. Could you please tell me how to do that (show me some existing usage in itests)?

Reply:
As suggested, please extract discussed methods into some util or other suitable class into maven.reddeeer and export this.

Reply:
I have created MavenProjectHelper class for maven utility methods.